src/clip.sol

```diff
@@ -124,6 +130,11 @@ contract Clipper {
         locked = 0;
     }
 
+    modifier isStopped(uint256 level, bool which) {
```

Comment:
I really don't think the second arg is necessary now that we have the levels...we're just passing it as true everywhere anyway.

Reply:
Yeah so with yank it was false. If we ever wanted a yank where we were say it has to be 3 or more then we want it. I was unsure on whether or not I wanted to keep it

Reply:
I would keep things simple--there's no need for it right now, it makes the code harder to read/less gas efficient, and we can add it back if we have to. `yank` could always implement some more custom condition without a modifier as a special case.

Reply:
Sounds good will pull it